04-spatial-operations.Rmd

```diff
@@ -559,18 +570,20 @@ The resulting `nz_avheight` objects have the same geometry as the aggregating ob
 Note that regions containing no points have an associated `elevation` value of `NA`.
 For aggregating operations which also create new geometries, see section \@ref(geometry-unions).
 
+<!-- One should avoid to have a single subsection, here 4.2.5.1. I would delete the heading, then spatial congruence just becomes another paragraph of spatial aggregation -->
 #### Spatial congruence and areal interpolation
 
 Spatial congruence is an important concept related to spatial aggregation.
-An *aggregating object* object (which we will refer to as `y`, representing the buffer object in the previous section) is *congruent* with the target object (`x`, representing the countries in the previous section) if the two objects have shared borders.
-Often this is the case for administrative boundary data, whereby the larger units (e.g., Middle Layer Super Output Areas in the UK or districts in many other European countries) are composed of many smaller units (Output Areas in this case, see [ons.gov.uk](https://www.ons.gov.uk/methodology/geography/ukgeographies/censusgeography) for further details or municipalities in many other European countries).
+An *aggregating object* object (which we will refer to as `y`) is *congruent* with the target object (`x`) if the two objects have shared borders.
+Often this is the case for administrative boundary data, whereby the larger units (e.g., Middle Layer Super Output Areas in the UK or districts in many other European countries) are composed of many smaller units (Output Areas in the UK, see [ons.gov.uk](https://www.ons.gov.uk/methodology/geography/ukgeographies/censusgeography) for further details or municipalities in many other European countries).
 
 *Incongruent* aggregating objects, by contrast, do not share common borders with the target [@qiu_development_2012].
 This is problematic for spatial aggregation (and other spatial operations) illustrated in Figure \@ref(fig:areal-example).
 Areal interpolation can help to alleviate this issue.
 It helps to transfer data from one set of areal units to another.
-A number of algorithms have been developed for areal interpolation, including area weighted and pycnophylactic interpolation methods task [@tobler_smooth_1979].
+A number of algorithms have been developed for areal interpolation, including area weighted and pycnophylactic interpolation methods [@tobler_smooth_1979].
 
+<!-- somehow the incongruent borders are hardly to discern in the right figure, maybe increasing the figure size could help -->
```

Comment:
somehow the incongruent borders are hardly to discern in the right figure, maybe increasing the figure size could help (though the figure is really nice!)

Reply:
Good point - much better now I think. Made it larger and added a border: 
![image](https://user-images.githubusercontent.com/1825120/34296920-fd1f4962-e70c-11e7-8154-1dc7da315833.png)

Reply:
In this commit: https://github.com/Robinlovelace/geocompr/commit/b7f9d2b50cbf34fdd511ae73f38720541cb2d062

Reply:
👍